backend/log.go

```diff
@@ -6,4 +6,4 @@ import (
 
 // Logger is the default logger instance. This can be used directly to log from
 // your plugin to grafana-server with calls like backend.Logger.Debug(...).
-var Logger log.Logger = log.New()
+var Logger log.Logger = log.DefaultLogger
```

Comment:
Shouldn't we remove this? Seems redundant.

Reply:
@bergquist Removing backend.Logger would change the API in a breaking fashion, in that clients would have to use backend.log.DefaultLogger instead.

Reply:
Ofc... I think I need to write a plugin to acknowledge the fact that the plugin-sdk is in use..